docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -41,42 +41,22 @@ Attack, Weapon Two:
         - Combat
         - Weapon Attacks
     Description: Your skills are sharpening, just like your weapon. 
-    Mechanic: Your Attacks as Major Actions now add an additional +1 to damage. 
+    Mechanic: Your Weapon Attack now deals an additional +1 damage. 
     Type: Major
-    XP: 1
-    Prereq:
-        Skill: 
-            - Finesse > 0
-            - Brute > 0
-        Power: Attack, Weapon
-        Level: 
-            Level > 2
-
-Attack, Weapon Three: 
-    Category: 
-        - Combat
-        - Weapon Attacks
-    Description: Your basic weapons training is complete. 
-    Mechanic: Your Attacks as Major Actions now add an additional +2 to damage. 
-    Type: Major
-    XP: 1
+    XP: 2
     Prereq:
-        Role: 
-            - Martial 
-            - Defender
         Skill: 
             - Finesse > 0
             - Brute > 0
         Power: Attack, Weapon
-        Level: 
-            Level > 4
+        Level: 3
 
 Attack, Charge:
     Category:
         - Combat
         - Weapon Attacks
     Description: Your weapon training allows you to throw your weight into an enemy, heightening your damage.
-    Mechanic: You must move 3 spaces before using this Power. You add +1 or 2 to the damage on a successful Weapon Attack. Power Points are still expended on a miss
+    Mechanic: You must move 3 spaces before using this Power. You add +1 or +2 to the damage on a successful Weapon Attack. Power Points are still expended on a miss
```

Comment:
I think it should be assumed 'choose PP on cast of Power' in general for any power with an optional cast, so maybe there's somewhere in the guide we would put that rather than at the Power level.